.github/workflows/test.yml

```diff
@@ -10,11 +10,14 @@ jobs:
   test-linux:
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v5
         with:
           fetch-depth: 0
+          # test that PAT works
```

Comment:
I.e. a different token (presumably a per-job one) is used by default, but explicitly using the one from `secrets` ensures that when it expires, jobs will fail prior to merge?